packages/itwinui-react/src/core/Stepper/StepperStep.test.tsx

```diff
@@ -131,6 +131,43 @@ describe('Stepper step (default)', () => {
     expect(container.querySelector('.iui-stepper-track-content')).toBeTruthy();
   });
 
+  it('should correctly pass custom className through stepProps, trackContentProps, circleProps, and nameProps', () => {
```

Comment:
Make sure a similar test is written for Workflow Diagram Step

Reply:
gotcha, wip